index.html

```diff
@@ -21,11 +21,6 @@
                     <section class="selections">
                         <div class="selections__parks">
                             <label for="parkSelect">Choose a park</label>
```

Comment:
would remove <div class="selections__parks"> and move it to your render functions so all that is left in the html is the selections section when all the selects are done